src/main/java/com/timgroup/statsd/DirectStatsDClient.java

```diff
@@ -0,0 +1,46 @@
+package com.timgroup.statsd;
+
+/**
+ * DirectStatsDClient is an experimental extension of {@link StatsDClient} that allows for direct access to some
+ * dogstatsd features.
+ *
+ * <p>It is not recommended to use this client in production. This client might allow you to take advantage of
+ * new features in the agent before they are released, but it might also break your application.
+ */
+public interface DirectStatsDClient extends StatsDClient {
+
+    /**
+     * Records values for the specified named distribution.
+     *
+     * <p>The method doesn't take care of breaking down the values array if it is too large. It's up to the caller to
+     * make sure the size is kept reasonable.</p>
+     *
+     * <p>This method is a DataDog extension, and may not work with other servers.</p>
+     *
+     * <p>This method is non-blocking and is guaranteed not to throw an exception.</p>
```

Comment:
I have kept the Javadoc consistant with the on `recordDistributionValue`:

```java
 /**
     * Records a value for the specified named distribution.
     *
     * <p>This method is a DataDog extension, and may not work with other servers.</p>
     *
     * <p>This method is non-blocking and is guaranteed not to throw an exception.</p>
     *
     * @param aspect
     *     the name of the distribution
     * @param value
     *     the value to be incorporated in the distribution
     * @param tags
     *     array of tags to be added to the data
     */
    void recordDistributionValue(String aspect, double value, String... tags);
```

So indeed it could block when using the blocking queue but as the `recordDistributionValue` and other methods in `StatsDClient`, isn't it? So do you prefer consistency with existing Javadoc or correctness?

Reply:
Ha, good point. Let's go with consistency for now then.